geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/conf/QueryProperties.scala

```diff
@@ -22,6 +22,7 @@ object QueryProperties {
 
   // rough upper limit on the number of ranges we will generate per query
   val ScanRangesTarget: SystemProperty = SystemProperty("geomesa.scan.ranges.target", "2000")
+  val ScanSeek        : SystemProperty = SystemProperty("geomesa.scan.seek", "true")
```

Comment:
Is this documented as a "system property"?  (I'm vaguely confused about how all the settings are documented, etc.)

Reply:
yep, right here: https://github.com/locationtech/geomesa/pull/2820/files#diff-9e613b450b11691f7415f85f61ed549cceddcc634eb6d08add885189b143ec9fR286